test/snapshot/class.test.js

```diff
@@ -129,15 +129,20 @@ describe("Test classes", function () {
     ).toMatchSnapshot();
   });
 
-  it("Test class union properties", function () {
-    expect(
-      parser.parseEval(`
-      class Test {
-        static int|float $foo;
-        private ?Foo|Bar $bar;
-        public Repo|string|null $a;
-      }`)
-    ).toMatchSnapshot();
+  it("Test promoted class properties php 8", function () {
```

Comment:
@czosel  seems that `it("Test class union properties", function () {` was removed, is there a reason ?